js/components/common/Disclaimer.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+/**
+ * A React component representing the disclaimer popup
+ */
+export default class Disclaimer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            dontShowDisclaimer: localStorage.getItem('DontShowDisclaimer')
+        }
+    }
+
+    handleClose = () => {
```

Comment:
Avoid modifying the DOM directly, since we want to use React to do this for us. This method should set the "hide disclaimer" state to `true`. You don't need the `dismiss` class or styling, I don't think the transition is worth it.